docs/solidity-by-example.rst

```diff
@@ -645,4 +645,499 @@ Safe Remote Purchase
 Micropayment Channel
 ********************
 
-To be written.
+Creating and verifying signatures
```

Comment:
Perhaps it would be good to have 1-2 sentences before the first subsection to explain the individual components of a micropayment channel, i.e. what is the perpose of the various subsections and what are we going to do in the "micropayment channel" section?